src/_sass/about.scss

```diff
@@ -0,0 +1,18 @@
+.bio {
```

Comment:
This isn't specific to _this_ line of CSS but I noticed on the about page, the bullet indentation in the box is a bit strange now:

<img width="845" alt="about page bullet point alignment" src="https://user-images.githubusercontent.com/841263/139948427-16fd087f-f04f-4bc9-b5e9-133ce0f3ea7a.png">

Reply:
Oh yeah. By default, I made it so that <ul> has no spacing around it, which works better when we use <ul> for navigational elements. But it messed up layout on some random components. Thanks for catching this one! I fixed it, and will keep on the lookout for others that need help.